<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8VF1PVM" failures="1" tests="9" name="Test" time="0.101" errors="0" timestamp="2022-10-28T12:49:18 IST">
  <testcase classname="com.failedtestcase.FailedTest2" name="test" time="0.015">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="com.failedtestcase.FailedTest2" name="test" time="0.002">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="com.failedtestcase.FailedTest2" name="test" time="0.002">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="com.failedtestcase.FailedTest2" name="test" time="0.002">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="com.failedtestcase.FailedTest2" name="test" time="0.002">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="com.failedtestcase.FailedTest2" name="test" time="0.002">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="com.failedtestcase.FailedTest2" name="test" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [A] but found [a]">
      <![CDATA[java.lang.AssertionError: expected [A] but found [a]
at com.failedtestcase.FailedTest2.test(FailedTest2.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase classname="com.failedtestcase.FailedTest2" name="test2" time="0.001"/>
  <testcase classname="com.failedtestcase.FailedTest2" name="test3" time="0.001"/>
</testsuite> <!-- Test -->
